docs/source/models.md

```diff
@@ -140,19 +146,19 @@ If there is only one _current owner_ the fulfillment will be a single signature
 }
 ```
 
-- `fid`: Fulfillment index. It matches a `cid` in the conditions with a new _crypto condition_ that the new owner(s)
-need to fulfill to spend this digital asset
-- `current_owners`: Public key of the current owner(s)
+- `fid`: Fulfillment index. It matches a `cid` in the conditions with a new _crypto condition_ that the new owner
+  needs to fulfill to spend this asset.
+- `current_owners`: A list of public keys of the current owners; in this case it has just one public key.
 - `fulfillment`:
```

Comment:
fulfillment: a cryptoconditions URI that encodes the cryptographic fulfillments like signatures and others, see crypto-conditions